### PR TITLE
Make registration checkbox easier to click

### PR DESCRIPTION
**Pull request by kaihendry**
Reference: https://github.com/unee-t/frontend/issues/405

Also made text less verbose.

### CODE REVIEW

File: imports/ui/signup/signup.jsx [hunk 1]
```diff
@@ -103,10 +103,7 @@ export class SignupPage extends Component {
           <div className='f7 gray mt3 lh-copy'>
             <label className='pa0 ma0 lh-copy f6 pointer mid-gray'>
               <input type='checkbox' checked={termsAgreement} onChange={this.handleTermCheckbox} />
-            </label> By signing up, you agree to our
-            <a className='link bondi-blue fw8' href='https://unee-t.com/privacy-and-terms/'> Terms of Service</a> and
-            <a className='link bondi-blue fw8' href='https://unee-t.com/privacy-and-terms/'> Privacy Policy</a> and
-            allow Unee-t to send you notifications and emails about your cases</div>
+            By signing up, you agree to our <a className='link bondi-blue fw8' target='_blank' href='https://unee-t.com/privacy-and-terms/'>Terms of Service &amp; Privacy Policy</a></label></div>
```
Comment (nbiton) [inline imports/ui/signup/signup.jsx]: Can you drop a few CRs in there? :laughing:

Comment (nbiton) [inline imports/ui/signup/signup.jsx]: Can you drop a few CRs in there? laughing

**Comment by franck-boullier** [comment]
This looks good but aybe we can add a space between the checkbox and the text to make it better

image

**Comment by nbiton** [comment]
oh, can you please use https://v0.material-ui.com/#/components/checkbox instead of that checkbox?

**Pull request merged**